source/level.js

```diff
@@ -138,23 +138,51 @@ export class Level extends Phaser.Scene{
 	        		level: level,
 	        		type: type
 	        	});
+	        	this.characterData.numberOfAutoClickers++;
 	        	this.autoClickers.push(autoClicker);
                 this.updateAutoClickerDPS(dps);
     		}       	
         });
 
         // Create kill quest
-        this.killQuestText = this.add.text(530, 270, this.enemiesKilled + "/" + this.killQuest + " " + this.enemy.name + "s killed", {fill: 'white'}).setDepth(3);
+        this.killQuestText = this.add.text(530, 270, this.characterData.enemiesKilled + "/" + this.killQuest + " " + this.enemySettings.name + "s killed", {fill: 'white'}).setDepth(3);
         this.questCompleteText = this.add.text(530, 290, 'Quest complete!', {fill: 'white'}).setDepth(3);
-        this.questCompleteText.visible = false;
+
+        // If level quest has already been completed
+        if (!this.characterData.levelCompletions.lumbridge) {
+        	this.questCompleteText.visible = false;
+        }
 
         // Show stats
         let statColor = 'white';
-        this.enemiesKilledText = this.add.text(20, 60, "Enemies killed: " + this.enemiesKilled, {fill: statColor}).setDepth(3);
-        this.timesClickedText = this.add.text(20, 75, "Times clicked: " + this.timesClicked, {fill: statColor}).setDepth(3);
-        this.damageByClickingText = this.add.text(20, 90, "Damage done by clicking: " + this.damageByClicking, {fill: statColor}).setDepth(3);
-        this.damageByAutoClickText = this.add.text(20, 105, "Damage done by autoclickers: " + this.damageByAutoClick, {fill: statColor}).setDepth(3);
+        this.enemiesKilledText = this.add.text(20, 60, "Enemies killed: " + this.characterData.enemiesKilled, {fill: statColor}).setDepth(3);
+        this.timesClickedText = this.add.text(20, 75, "Times clicked: " + this.characterData.timesClicked, {fill: statColor}).setDepth(3);
+        this.damageByClickingText = this.add.text(20, 90, "Damage done by clicking: " + this.characterData.damageByClicking, {fill: statColor}).setDepth(3);
+        this.damageByAutoClickText = this.add.text(20, 105, "Damage done by autoclickers: " + this.characterData.damageByAutoClick, {fill: statColor}).setDepth(3);
         this.autoClickDpsText = this.add.text(20, 120, "AutoClicker DPS: " + this.autoClickDps, {fill: statColor}).setDepth(3);
+    
+        // Unpause autoclickers
+        for (let i = 0; i < this.autoClickers.length; i++) {
+            this.autoClickers[i].pause = false;
+        }
+       
+        // Re-add autoclickers from cookies on first load
+        if (this.characterData.hasCookies && this.autoClickers.length == 0){
+            for (let i = 0; i < this.characterData.numberOfAutoClickers; i++) {
+                let dps = 5;
+                let level = 1;
+                let type = 'Hired Bowman';
```

Comment:
nice. this gives me ideas for autoclickers like "join a clan" - "get a gf"

Reply:
Gotta carve and forge that diamond ring on your own to show you truly care